aspnet/fundamentals/error-handling.rst

```diff
@@ -1,8 +1,181 @@
-.. include:: /../common/stub-topic.txt
+Error Handling
+==============
+By `Steve Smith`_
 
-|stub-icon| Error Handling
-==========================
+When errors occur in your ASP.NET app, you can handle them in a variety of ways, as described in this article.
```

Comment:
Changed most to Exception, except in a few places where I thought Error still appropriate.